circuitpython_community_library_list.md

```diff
@@ -165,3 +165,4 @@ c* [CircuitPython_RotarySelect](https://github.com/FoamyGuy/CircuitPython_Rotary
 * [RS485-Wrapper](https://github.com/facts-engineering/CircuitPython_rs485_wrapper.git) A CircuitPython helper library to convert a UART object into an RS485 object.
 * [Cirque Pinnacle](https://github.com/2bndy5/CircuitPython_Cirque_Pinnacle) A driver library to use circular trackpads (as seen in the Steam controller and HTC Vive VR controllers) empowered with Cirque's 1CA027 ASIC (surnamed "Pinnacle"). More information can be found in the [documentation](https://circuitpython-cirque-pinnacle.rtfd.io).
 * [Embedded HTTPServer](https://github.com/bablokb/circuitpython-ehttpserver) An efficient and slim HTTP-server (fork of biplane)
+* [MH-Z19x](https://github.com/bablokb/circuitpython-mhz19.git) Driver for the MH-Z19x CO2-sensors
```

Comment:
This can go up here in the drivers section instead of helpers. https://github.com/bablokb/CircuitPython_Community_Bundle/blob/mhz19/circuitpython_community_library_list.md?plain=1#L71-L72 

I think they are intended to be alphabetical, so between M5stack.. and Mindwidgets.. for this one.